samples/bluetooth/broadcast_audio_sink/src/main.c

```diff
@@ -712,7 +769,8 @@ int main(void)
 			}
 		}
 
-		printk("Scanning for broadcast sources\n");
+		printk("Scanning for broadcast source " CONFIG_BT_TARGET_BROADCAST_NAME "\n");
```

Comment:
Shouldn't this only be printed if `CONFIG_BT_TARGET_BROADCAST_NAME != ""`?

Reply:
Yes, however if CONFIG_BT_TARGET_BROADCAST_NAME is an empty string, the output still makes sense and we avoids an if-check. But I also agree with your change  :) Thoughts?

Reply:
Ah, yeah, you are correct. It is fine to keep as is (the extra spaces before the newline doesn't matter either).

Reply:
I ended up implementing your suggestion and re-phrased the output to avoid ambiguity for users.